_posts/jared_baker/2018-12-15-Bluestore.md

```diff
@@ -0,0 +1,171 @@
+---
+layout: post
+title:  "Migrating 8 PB of data from Filestore to Bluestore"
+breadcrumb: true
+author: jared_baker
+date: 2018-12-15
+categories: jared_baker
+tags:
+    - ceph
+    - linux
+    - bluestore
+    - tutorial
+teaser:
+    info: How we migrated an 8 PB cluster from Filestore to Bluestore
+    image: jared_baker/bluestore/Ceph_Logo_Standard_RGB_120411_fa.png
+header:
+    version: small
+    title: Software Engineering Blog
+    image: jared_baker/le-logo-standard.png
+    icon: icon-blog
+---
+## Introduction
+The Cancer Genome Collaboratory is a Cloud computing environment designed for cancer research and an official repository of 950 TB (and growing) worth of whole genomes for the [ICGC project](https://dcc.icgc.org/repositories). We have used OpenStack and Ceph to provide an extremely scalable and reliable infrastructure. Maintaining the pieces namely through upgrades is our most important and time consuming task in operating a cloud like this. Whether you are chasing new features, bug fixes, security patches or supportability it seems like we are always planning for the next upgrade.
```

Comment:
Consider re-wording as _"...official repository of **950 TB worth (and growing)**..."_